projects/1.16/assets/dustrial-decor/dustrial_decor/lang/zh_cn.json

```
@@ -1 +1,114 @@
-{}
\ No newline at end of file
+{
+    "item.dustrial_decor.industrial_iron_billet": "工业铁锭",
+    "block.dustrial_decor.industrial_iron_block": "工业铁块",
+    "block.dustrial_decor.industrial_iron_pillar": "工业铁柱",
+    "block.dustrial_decor.bolted_industrial_iron_block": "螺栓工业铁块",
+    "block.dustrial_decor.bolted_industrial_iron_slab": "螺栓工业铁台阶",
+    "block.dustrial_decor.bolted_industrial_iron_vertical_slab": "螺栓工业铁竖直台阶",
+    "block.dustrial_decor.bolted_industrial_iron_stairs": "螺栓工业铁楼梯",
+    "block.dustrial_decor.industrial_iron_trapdoor": "工业铁活板门",
+    "block.dustrial_decor.industrial_iron_door": "工业铁门",
+    "item.dustrial_decor.cast_iron_billet": "铸铁锭",
+    "block.dustrial_decor.cast_iron_block": "铸铁块",
+    "block.dustrial_decor.cast_iron_pillar": "铸铁柱",
+    "block.dustrial_decor.cast_iron_bricks": "铸铁砖",
+    "block.dustrial_decor.cast_iron_brick_slab": "铸铁砖台阶",
+    "block.dustrial_decor.cast_iron_brick_vertical_slab": "铸铁砖竖直台阶",
+    "block.dustrial_decor.cast_iron_brick_stairs": "铸铁砖楼梯",
+    "block.dustrial_decor.engraved_cast_iron_block": "雕刻铸铁块",
+    "block.dustrial_decor.cast_iron_balustrade": "铸铁栏杆",
+    "block.dustrial_decor.chain_link_fence": "链环围栏",
+    "block.dustrial_decor.barbed_chain_link_fence": "带刺链环围栏",
+    "block.dustrial_decor.barbed_iron_bars": "带刺铁栏杆",
+    "block.dustrial_decor.chain_trapdoor": "链环活板门",
+    "block.dustrial_decor.chain_door": "链环门",
+    "block.dustrial_decor.padded_block": "填充方块",
+    "block.dustrial_decor.mini_padded_block": "迷你填充方块",

```

```suggestion
    "block.dustrial_decor.mini_padded_block": "小方格填充方块",
```

这里**谨慎**修改
游戏内是有小方格图案的方块，看起来像是由小型的填充方块组成的，但`迷你填充方块`可能有歧义